src/main/java/com/mitchej123/hodgepodge/config/FixesConfig.java

```diff
@@ -489,6 +489,10 @@ public class FixesConfig {
     @Config.DefaultBoolean(true)
     public static boolean fixExtraUtilitiesHealingAxeHeal;
 
+    @Config.Comment("Fixes Extra Utilities chests not updating comparators Redstone signal when their inventory changes")
```

Comment:
This cleans up the writing a little:
```suggestion
    @Config.Comment("Fix Extra Utilities chests not updating comparator redstone signals when their inventories change")
```